ert_shared/main.py

```diff
@@ -196,10 +206,17 @@ def get_ert_parser(parser=None):
         "Assimilation Ensemble Smoother will half the weight applied to the "
         "Observation Errors from one iteration to the next across 4 iterations.",
     )
+    es_mda_parser.add_argument(
+        "--current-case",
+        type=valid_name,
+        required=False,
+        help="This is the name of the case where the results for the simulation "
```

Comment:
Up for debate, but maybe we can just write "_Name of the case where..._"

Reply:
:+1: